housekeeping-vacuum-tool/pom.xml

```diff
@@ -58,7 +58,7 @@
     <dependency>
       <groupId>com.hotels</groupId>
       <artifactId>hcommon-hive-metastore</artifactId>
-      <version>1.2.0</version>
+      <version>1.3.1-SNAPSHOT</version>
```

Comment:
This will become 1.4.0 but let's worry about that once the other PR is merged and released.